package.json

```diff
@@ -28,6 +28,6 @@
     "stylelint-order": "^6.0.4",
     "turbo": "^2.3.3",
     "typescript": "^5.7.3",
-    "vitest": "^3.0.2"
+    "vitest": "^2.1.8"
```

Comment:
Downgraded back to 2.x; was seeing some uncaught processes in CI. Will update a few patches later